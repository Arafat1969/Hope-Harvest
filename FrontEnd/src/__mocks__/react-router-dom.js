// src/__mocks__/react-router-dom.js
const React = require("react");

function makeAnchor({ children, to, className, ...rest }) {
  // ensure className is a string
  const cn = Array.isArray(className) ? className.join(" ") : className;
  return (
    <a href={to} className={cn} {...rest}>
      {children}
    </a>
  );
}

module.exports = {
  BrowserRouter: ({ children }) => <>{children}</>,
  MemoryRouter: ({ children }) => <>{children}</>,
  Routes: ({ children }) => <>{children}</>,
  Route: ({ element }) => element,

  Link: (props) => makeAnchor(props),
  NavLink: (props) => makeAnchor(props),

  useNavigate: () => () => {},
  useLocation: () => ({ pathname: "/" }),
};
