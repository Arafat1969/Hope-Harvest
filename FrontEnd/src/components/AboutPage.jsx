import { useState } from 'react';
import { Link } from 'react-router-dom';
import styles from './AboutPage.module.css';

const AboutPage = () => {
  const [activeItem, setActiveItem] = useState('introduction');

  const menuItems = [
    { id: 'introduction', label: 'Introduction', icon: 'fas fa-home' },
    { id: 'principles', label: 'Principles and Norms', icon: 'fas fa-balance-scale' },
    { id: 'goals', label: 'Goals and Objectives', icon: 'fas fa-bullseye' },
    { id: 'activities', label: 'Activities', icon: 'fas fa-hands-helping' },
    { id: 'sources', label: 'Sources of funds', icon: 'fas fa-money-bill-wave' },
    { id: 'expenditure', label: 'Expenditure policy', icon: 'fas fa-chart-pie' },
    { id: 'achievements', label: 'Achievements', icon: 'fas fa-trophy' },
  ];
  
  const contentData = {
    introduction: {
      icon: 'fas fa-seedling',
      items: [
        'Hope Harvest was established in 2025 as a charitable foundation with the mission of "Sowing Seeds of Hope, Reaping Change."',
        'Our foundation focuses on providing sustainable relief, empowering communities, and creating long-term positive impact across Bangladesh.',
        'Founded by a group of dedicated community leaders who recognized the need for transparent and efficient charitable services.',
        'We operate through three key service areas: User Authentication & Management, Donation & Payment Services, and Event & Volunteer Coordination.',
        'Our digital platform enables secure donations, transparent fund management, volunteer coordination, and efficient event organization.',
        'Through campaigns and events, we\'ve touched the lives of thousands across various districts, focusing on areas like disaster relief, education, and healthcare.',
        'We maintain complete transparency in our operations, with detailed tracking of all donations and their impacts.'
      ]
    },
    principles: {
      icon: 'fas fa-balance-scale',
      items: [
        { title: 'Transparency', desc: 'We maintain complete openness about our finances, operations, and impact. Donors can track their contributions from donation to implementation.' },
        { title: 'Accountability', desc: 'We hold ourselves accountable to donors, beneficiaries, and the public through regular reporting and independent audits.' },
        { title: 'Efficiency', desc: 'We minimize administrative costs (5-10%) to ensure maximum impact from every donation we receive.' },
        { title: 'Inclusivity', desc: 'We serve all communities regardless of religion, ethnicity, gender, or political affiliation.' },
        { title: 'Sustainability', desc: 'Our projects aim for long-term impact and self-sufficiency rather than creating dependency.' },
        { title: 'Dignity', desc: 'We respect the dignity of beneficiaries and involve them in decision-making processes.' },
      ]
    },
    goals: { icon: 'fas fa-bullseye', items: [ { title: 'Relief Provision', desc: 'Deliver timely and effective relief during emergencies and natural disasters.' }, { title: 'Poverty Alleviation', desc: 'Create sustainable pathways out of poverty through education, skills training, and economic opportunities.' } /*...more*/ ]},
    activities: { icon: 'fas fa-hands-helping', items: [ { title: 'Disaster Relief', desc: 'Providing immediate assistance during floods, cyclones, and other disasters with food, shelter, and medical aid.' }, { title: 'Winter Relief', desc: 'Distribution of blankets, warm clothing, and heaters to vulnerable communities during winter months.' } /*...more*/ ]},
    sources: { icon: 'fas fa-money-bill-wave', items: [ 'The journey begins with the property and funds purchased with donations from the founding members.', 'One-time and regular donations from members and supporters through our secure online platform.' /*...more*/ ]},
    expenditure: { icon: 'fas fa-chart-pie', items: [ { title: 'Campaign-Specific Allocation', desc: 'Donations to specific campaigns are strictly used for those campaigns, with transparent usage reporting.' }, { title: 'Administrative Cap', desc: 'We maintain a strict cap of 5-10% on administrative expenses to ensure most funds reach beneficiaries.' } /*...more*/ ]},
    achievements: { icon: 'fas fa-trophy', items: [ { title: 'Relief Impact', desc: 'Provided emergency relief to over 25,000 people affected by floods, cyclones, and other disasters since 2015.' }, { title: 'Winter Campaigns', desc: 'Distributed warm clothing and blankets to 15,000+ families in northern Bangladesh over the past 5 years.' } /*...more*/ ]},
  };

  const renderContent = () => {
    const currentContent = contentData[activeItem];
    if (!currentContent) return null;

    const isSimpleList = typeof currentContent.items[0] === 'string';

    return (
      <div>
        <div className="text-center mb-4">
          <i className={`${currentContent.icon} fa-3x ${styles.contentIcon}`}></i>
        </div>
        <ul className="list-unstyled">
          {currentContent.items.map((item, index) => (
            <li key={index} className={styles.listItem}>
              <span className={styles.bulletPoint}>ðŸŸ¢</span>
              {typeof item === 'string' ? item : (
                <p>
                  <span className={styles.strongText}>{item.title}:</span> {item.desc}
                </p>
              )}
            </li>
          ))}
        </ul>
      </div>
    );
  };

  return (
    <div className={styles.mainContainer}>
      <div className="container">
        {/* Hero Section */}
        <div className={styles.heroSection}>
          <i className={`fas fa-heart ${styles.heroPattern}`}></i>
          <div className="row align-items-center">
            <div className="col-md-8">
              <h1 className="display-4 fw-bold">About Hope Harvest</h1>
              <p className="lead">
                Sowing Seeds of Hope, Reaping Change - Learn about our mission, values, and impact in creating sustainable positive change across Bangladesh.
              </p>
            </div>
            <div className="col-md-4 text-center">
              <div className="row g-3">
                <div className="col-6">
                  <div className={styles.statsCard}>
                    <h4>2025</h4>
                    <p className="mb-0">Established</p>
                  </div>
                </div>
                <div className="col-6">
                  <div className={styles.statsCard}>
                    <h4>25K+</h4>
                    <p className="mb-0">Lives Impacted</p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>

        {/* This `align-items-stretch` class is the key to equal height columns */}
        <div className="row align-items-stretch">
          {/* Sidebar */}
          <div className="col-md-4">
            <div className={styles.sidebarContainer}>
              <h4 className={styles.sidebarTitle}>
                <i className="fas fa-book-open me-2"></i>
                Explore Our Story
              </h4>
              <ul className="list-unstyled">
                {menuItems.map(item => (
                  <li
                    key={item.id}
                    className={`${styles.sidebarItem} ${activeItem === item.id ? styles.activeItem : ''}`}
                    onClick={() => setActiveItem(item.id)}
                  >
                    <div className={styles.iconContainer}>
                      <i className={item.icon}></i>
                    </div>
                    {item.label}
                  </li>
                ))}
              </ul>
            </div>
          </div>

          {/* Content */}
          <div className="col-md-8">
            <div className={styles.contentContainer}>
              <h2 className={styles.contentTitle}>
                {menuItems.find(item => item.id === activeItem)?.label}
              </h2>
              <div className={styles.contentTitleUnderline}></div>
              {renderContent()}
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default AboutPage;
