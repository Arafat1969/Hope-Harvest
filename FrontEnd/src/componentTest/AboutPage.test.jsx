import React from "react";
import { render, screen, fireEvent, within } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import AboutPage from "../components/AboutPage";

describe("AboutPage Component", () => {
  const renderComponent = () => {
    return render(
      <MemoryRouter>
        <AboutPage />
      </MemoryRouter>
    );
  };

  beforeEach(() => {
    // Clear any previous renders - use Testing Library's cleanup instead
  });

  it("renders default introduction content (h2 only)", () => {
    renderComponent();
    const introHeading = screen.getByRole("heading", {
      level: 2,
      name: "Introduction",
    });
    expect(introHeading).toBeInTheDocument();
  });

  it("displays exactly seven sidebar menu items", () => {
    renderComponent();
    
    // Find the sidebar by its heading first, then find the list
    const sidebarHeading = screen.getByRole("heading", { 
      level: 4, 
      name: /Explore Our Story/i 
    });
    expect(sidebarHeading).toBeInTheDocument();
    
    // Find all lists and get the first one (sidebar menu)
    const menuLists = screen.getAllByRole("list");
    const menuList = menuLists[0]; // First list is the sidebar menu
    const items = within(menuList).getAllByRole("listitem");
    
    expect(items).toHaveLength(7);
    expect(items.map((li) => li.textContent.trim())).toEqual([
      "Introduction",
      "Principles and Norms",
      "Goals and Objectives",
      "Activities",
      "Sources of funds and income",
      "Expenditure policy",
      "Achievements",
    ]);
  });

  it("switches content when menu items are clicked", () => {
    renderComponent();
    
    // Find the first list (sidebar menu)
    const menuLists = screen.getAllByRole("list");
    const menuList = menuLists[0];
    const normsItem = within(menuList).getByText("Principles and Norms");
    fireEvent.click(normsItem);

    // Assert the main content heading updates
    expect(
      screen.getByRole("heading", {
        level: 2,
        name: "Principles and Norms",
      })
    ).toBeInTheDocument();

    // Assert that the first bullet includes "Transparency:"
    expect(screen.getByText(/Transparency:/)).toBeInTheDocument();
  });

  it("displays hero statistics", () => {
    renderComponent();
    
    // Look for statistics text content
    expect(screen.getByText("2025")).toBeInTheDocument();
    expect(screen.getByText("25K+")).toBeInTheDocument();
  });

  it("displays hero section with correct title", () => {
    renderComponent();
    
    expect(
      screen.getByRole("heading", { level: 1, name: "About Hope Harvest" })
    ).toBeInTheDocument();
  });

  it("displays sidebar title correctly", () => {
    renderComponent();
    
    expect(
      screen.getByRole("heading", { level: 4, name: /Explore Our Story/i })
    ).toBeInTheDocument();
  });

  it("renders all menu items with icons", () => {
    renderComponent();
    
    // Find the first list (sidebar menu)
    const menuLists = screen.getAllByRole("list");
    const menuList = menuLists[0];
    
    const menuItems = [
      "Introduction",
      "Principles and Norms", 
      "Goals and Objectives",
      "Activities",
      "Sources of funds and income",
      "Expenditure policy",
      "Achievements"
    ];

    menuItems.forEach(item => {
      expect(within(menuList).getByText(item)).toBeInTheDocument();
    });
  });

  it("changes active state when clicking different menu items", () => {
    renderComponent();
    
    // Find the first list (sidebar menu)
    const menuLists = screen.getAllByRole("list");
    const menuList = menuLists[0];
    
    // Click on "Goals and Objectives"
    const goalsItem = within(menuList).getByText("Goals and Objectives");
    fireEvent.click(goalsItem);
    
    // Check that the content heading changed
    expect(
      screen.getByRole("heading", {
        level: 2,
        name: "Goals and Objectives",
      })
    ).toBeInTheDocument();
  });

  it("can navigate through all menu sections", () => {
    renderComponent();
    
    const menuLists = screen.getAllByRole("list");
    const menuList = menuLists[0];
    const menuSections = [
      { name: "Activities", heading: "Activities" },
      { name: "Achievements", heading: "Achievements" },
      { name: "Expenditure policy", heading: "Expenditure policy" }
    ];

    menuSections.forEach(section => {
      const menuItem = within(menuList).getByText(section.name);
      fireEvent.click(menuItem);
      
      expect(
        screen.getByRole("heading", {
          level: 2,
          name: section.heading,
        })
      ).toBeInTheDocument();
    });
  });

  it("displays correct initial active state", () => {
    renderComponent();
    
    // Check that Introduction is the default active section
    expect(
      screen.getByRole("heading", {
        level: 2,
        name: "Introduction",
      })
    ).toBeInTheDocument();
  });

  it("renders without crashing and has required structural elements", () => {
    renderComponent();
    
    // Check for structural elements that actually exist
    expect(screen.getByRole("heading", { level: 1 })).toBeInTheDocument();
    expect(screen.getByRole("heading", { level: 4, name: /Explore Our Story/i })).toBeInTheDocument();
    
    // Check that we have lists (both sidebar menu and content lists)
    const lists = screen.getAllByRole("list");
    expect(lists.length).toBeGreaterThanOrEqual(1);
  });

  it("has proper container structure", () => {
    renderComponent();
    
    // Check for Bootstrap container class
    const container = document.querySelector('.container');
    expect(container).toBeInTheDocument();
    
    // Check for row structure
    const rows = document.querySelectorAll('.row');
    expect(rows.length).toBeGreaterThan(0);
  });

  it("displays hero section with statistics", () => {
    renderComponent();
    
    // Check for hero section elements
    expect(screen.getByText("About Hope Harvest")).toBeInTheDocument();
    expect(screen.getByText("Established")).toBeInTheDocument();
    expect(screen.getByText("Lives Impacted")).toBeInTheDocument();
  });

  it("matches snapshot", () => {
    const { asFragment } = renderComponent();
    expect(asFragment()).toMatchSnapshot();
  });
});