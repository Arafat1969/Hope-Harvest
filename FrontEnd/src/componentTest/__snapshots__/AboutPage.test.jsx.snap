// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    style="min-height: 100vh; padding-top: 2rem; padding-bottom: 2rem;"
  >
    <div
      class="container"
    >
      <div
        style="border-radius: 20px; padding: 3rem 2rem; color: white; margin-bottom: 3rem; position: relative; overflow: hidden;"
      >
        <i
          class="fas fa-heart"
          style="position: absolute; top: -50px; right: -50px; opacity: 0.1; font-size: 15rem; transform: rotate(-15deg);"
        />
        <div
          class="row align-items-center"
        >
          <div
            class="col-md-8"
          >
            <h1
              class="display-4 fw-bold mb-4"
            >
              About Hope Harvest
            </h1>
            <p
              class="lead mb-4"
            >
              Sowing Seeds of Hope, Reaping Change - Learn about our mission, values, and impact in creating sustainable positive change across Bangladesh.
            </p>
          </div>
          <div
            class="col-md-4 text-center"
          >
            <div
              class="row"
            >
              <div
                class="col-6"
              >
                <div
                  style="color: white; border-radius: 15px; padding: 1.5rem; text-align: center; margin: 0.5rem; box-shadow: 0 4px 15px rgba(0,0,0,0.1);"
                >
                  <h4>
                    2025
                  </h4>
                  <p
                    class="mb-0"
                  >
                    Established
                  </p>
                </div>
              </div>
              <div
                class="col-6"
              >
                <div
                  style="color: white; border-radius: 15px; padding: 1.5rem; text-align: center; margin: 0.5rem; box-shadow: 0 4px 15px rgba(0,0,0,0.1);"
                >
                  <h4>
                    25K+
                  </h4>
                  <p
                    class="mb-0"
                  >
                    Lives Impacted
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
              
      <div
        class="row d-flex align-items-stretch"
        style="min-height: 700px;"
      >
        <div
          class="col-md-4"
        >
          <div
            style="background: white; border-radius: 15px; padding: 1.5rem; box-shadow: 0 8px 32px rgba(0,0,0,0.1); min-height: 700px; width: 100%;"
          >
            <h4
              style="color: rgb(46, 125, 50); font-weight: 700; margin-bottom: 1.5rem; text-align: center; font-size: 1.3rem;"
            >
              <i
                class="fas fa-book-open me-2"
              />
              Explore Our Story
            </h4>
            <ul
              class="list-unstyled"
            >
              <li
                class="list-group-item-action"
                style="cursor: pointer; transition: all 0.3s ease; border-radius: 10px; margin: 0.3rem 0px; padding: 1rem 1.2rem; font-size: 1rem; font-weight: 500; position: relative; overflow: hidden; color: white; transform: translateX(8px); box-shadow: 0 4px 15px rgba(76, 175, 80, 0.3);"
              >
                <div
                  style="width: 40px; height: 40px; border-radius: 50%; background: rgba(76, 175, 80, 0.1); display: inline-flex; align-items: center; justify-content: center; margin-right: 1rem; font-size: 1.2rem; color: rgb(76, 175, 80);"
                >
                  <i
                    class="fas fa-home"
                  />
                </div>
                Introduction
              </li>
              <li
                class="list-group-item-action"
                style="cursor: pointer; transition: all 0.3s ease; border: 1px solid #e9ecef; border-radius: 10px; margin: 0.3rem 0px; padding: 1rem 1.2rem; font-size: 1rem; font-weight: 500; position: relative; overflow: hidden; background-color: rgb(248, 249, 250); color: rgb(73, 80, 87);"
              >
                <div
                  style="width: 40px; height: 40px; border-radius: 50%; background: rgba(76, 175, 80, 0.1); display: inline-flex; align-items: center; justify-content: center; margin-right: 1rem; font-size: 1.2rem; color: rgb(76, 175, 80);"
                >
                  <i
                    class="fas fa-balance-scale"
                  />
                </div>
                Principles and Norms
              </li>
              <li
                class="list-group-item-action"
                style="cursor: pointer; transition: all 0.3s ease; border: 1px solid #e9ecef; border-radius: 10px; margin: 0.3rem 0px; padding: 1rem 1.2rem; font-size: 1rem; font-weight: 500; position: relative; overflow: hidden; background-color: rgb(248, 249, 250); color: rgb(73, 80, 87);"
              >
                <div
                  style="width: 40px; height: 40px; border-radius: 50%; background: rgba(76, 175, 80, 0.1); display: inline-flex; align-items: center; justify-content: center; margin-right: 1rem; font-size: 1.2rem; color: rgb(76, 175, 80);"
                >
                  <i
                    class="fas fa-target"
                  />
                </div>
                Goals and Objectives
              </li>
              <li
                class="list-group-item-action"
                style="cursor: pointer; transition: all 0.3s ease; border: 1px solid #e9ecef; border-radius: 10px; margin: 0.3rem 0px; padding: 1rem 1.2rem; font-size: 1rem; font-weight: 500; position: relative; overflow: hidden; background-color: rgb(248, 249, 250); color: rgb(73, 80, 87);"
              >
                <div
                  style="width: 40px; height: 40px; border-radius: 50%; background: rgba(76, 175, 80, 0.1); display: inline-flex; align-items: center; justify-content: center; margin-right: 1rem; font-size: 1.2rem; color: rgb(76, 175, 80);"
                >
                  <i
                    class="fas fa-hands-helping"
                  />
                </div>
                Activities
              </li>
              <li
                class="list-group-item-action"
                style="cursor: pointer; transition: all 0.3s ease; border: 1px solid #e9ecef; border-radius: 10px; margin: 0.3rem 0px; padding: 1rem 1.2rem; font-size: 1rem; font-weight: 500; position: relative; overflow: hidden; background-color: rgb(248, 249, 250); color: rgb(73, 80, 87);"
              >
                <div
                  style="width: 40px; height: 40px; border-radius: 50%; background: rgba(76, 175, 80, 0.1); display: inline-flex; align-items: center; justify-content: center; margin-right: 1rem; font-size: 1.2rem; color: rgb(76, 175, 80);"
                >
                  <i
                    class="fas fa-money-bill-wave"
                  />
                </div>
                Sources of funds and income
              </li>
              <li
                class="list-group-item-action"
                style="cursor: pointer; transition: all 0.3s ease; border: 1px solid #e9ecef; border-radius: 10px; margin: 0.3rem 0px; padding: 1rem 1.2rem; font-size: 1rem; font-weight: 500; position: relative; overflow: hidden; background-color: rgb(248, 249, 250); color: rgb(73, 80, 87);"
              >
                <div
                  style="width: 40px; height: 40px; border-radius: 50%; background: rgba(76, 175, 80, 0.1); display: inline-flex; align-items: center; justify-content: center; margin-right: 1rem; font-size: 1.2rem; color: rgb(76, 175, 80);"
                >
                  <i
                    class="fas fa-chart-pie"
                  />
                </div>
                Expenditure policy
              </li>
              <li
                class="list-group-item-action"
                style="cursor: pointer; transition: all 0.3s ease; border: 1px solid #e9ecef; border-radius: 10px; margin: 0.3rem 0px; padding: 1rem 1.2rem; font-size: 1rem; font-weight: 500; position: relative; overflow: hidden; background-color: rgb(248, 249, 250); color: rgb(73, 80, 87);"
              >
                <div
                  style="width: 40px; height: 40px; border-radius: 50%; background: rgba(76, 175, 80, 0.1); display: inline-flex; align-items: center; justify-content: center; margin-right: 1rem; font-size: 1.2rem; color: rgb(76, 175, 80);"
                >
                  <i
                    class="fas fa-trophy"
                  />
                </div>
                Achievements
              </li>
            </ul>
          </div>
        </div>
                  
        <div
          class="col-md-8"
        >
          <div
            style="background-color: rgb(255, 255, 255); border-radius: 15px; padding: 2.5rem; box-shadow: 0 8px 32px rgba(0,0,0,0.1); border: 1px solid rgba(76, 175, 80, 0.1); min-height: 700px; width: 100%;"
          >
            <h2
              style="color: rgb(46, 125, 50); font-weight: 700; font-size: 2rem; margin-bottom: 1rem; text-align: center; position: relative;"
            >
              Introduction
            </h2>
            <div
              style="width: 80px; height: 4px; margin: 0px auto 2rem; border-radius: 2px;"
            />
            <div>
              <div
                class="text-center mb-4"
              >
                <i
                  class="fas fa-seedling fa-3x"
                  style="color: rgb(76, 175, 80); margin-bottom: 1rem;"
                />
              </div>
              <ul
                class="list-unstyled"
              >
                <li
                  style="margin-bottom: 1.2rem; padding: 1rem; background-color: rgb(248, 249, 250); border-radius: 10px; border: 1px solid #e9ecef; transition: all 0.3s ease; position: relative; font-size: 1rem; line-height: 1.6;"
                >
                  <span
                    style="color: rgb(76, 175, 80); font-size: 1.2rem; margin-right: 0.8rem; display: inline-block; animation: pulse 2s infinite;"
                  >
                    🟢
                  </span>
                  Hope Harvest was established in 2025 as a charitable foundation with the mission of "Sowing Seeds of Hope, Reaping Change."
                </li>
                <li
                  style="margin-bottom: 1.2rem; padding: 1rem; background-color: rgb(248, 249, 250); border-radius: 10px; border: 1px solid #e9ecef; transition: all 0.3s ease; position: relative; font-size: 1rem; line-height: 1.6;"
                >
                  <span
                    style="color: rgb(76, 175, 80); font-size: 1.2rem; margin-right: 0.8rem; display: inline-block; animation: pulse 2s infinite;"
                  >
                    🟢
                  </span>
                  Our foundation focuses on providing sustainable relief, empowering communities, and creating long-term positive impact across Bangladesh.
                </li>
                <li
                  style="margin-bottom: 1.2rem; padding: 1rem; background-color: rgb(248, 249, 250); border-radius: 10px; border: 1px solid #e9ecef; transition: all 0.3s ease; position: relative; font-size: 1rem; line-height: 1.6;"
                >
                  <span
                    style="color: rgb(76, 175, 80); font-size: 1.2rem; margin-right: 0.8rem; display: inline-block; animation: pulse 2s infinite;"
                  >
                    🟢
                  </span>
                  Founded by a group of dedicated community leaders who recognized the need for transparent and efficient charitable services.
                </li>
                <li
                  style="margin-bottom: 1.2rem; padding: 1rem; background-color: rgb(248, 249, 250); border-radius: 10px; border: 1px solid #e9ecef; transition: all 0.3s ease; position: relative; font-size: 1rem; line-height: 1.6;"
                >
                  <span
                    style="color: rgb(76, 175, 80); font-size: 1.2rem; margin-right: 0.8rem; display: inline-block; animation: pulse 2s infinite;"
                  >
                    🟢
                  </span>
                  We operate through three key service areas: User Authentication & Management, Donation & Payment Services, and Event & Volunteer Coordination.
                </li>
                <li
                  style="margin-bottom: 1.2rem; padding: 1rem; background-color: rgb(248, 249, 250); border-radius: 10px; border: 1px solid #e9ecef; transition: all 0.3s ease; position: relative; font-size: 1rem; line-height: 1.6;"
                >
                  <span
                    style="color: rgb(76, 175, 80); font-size: 1.2rem; margin-right: 0.8rem; display: inline-block; animation: pulse 2s infinite;"
                  >
                    🟢
                  </span>
                  Our digital platform enables secure donations, transparent fund management, volunteer coordination, and efficient event organization.
                </li>
                <li
                  style="margin-bottom: 1.2rem; padding: 1rem; background-color: rgb(248, 249, 250); border-radius: 10px; border: 1px solid #e9ecef; transition: all 0.3s ease; position: relative; font-size: 1rem; line-height: 1.6;"
                >
                  <span
                    style="color: rgb(76, 175, 80); font-size: 1.2rem; margin-right: 0.8rem; display: inline-block; animation: pulse 2s infinite;"
                  >
                    🟢
                  </span>
                  Through campaigns and events, we've touched the lives of thousands across various districts, focusing on areas like disaster relief, education, healthcare, and community development.
                </li>
                <li
                  style="margin-bottom: 1.2rem; padding: 1rem; background-color: rgb(248, 249, 250); border-radius: 10px; border: 1px solid #e9ecef; transition: all 0.3s ease; position: relative; font-size: 1rem; line-height: 1.6;"
                >
                  <span
                    style="color: rgb(76, 175, 80); font-size: 1.2rem; margin-right: 0.8rem; display: inline-block; animation: pulse 2s infinite;"
                  >
                    🟢
                  </span>
                  We maintain complete transparency in our operations, with detailed tracking of all donations and their impacts.
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
    <style>
      
        @keyframes pulse {
          0% { transform: scale(1); }          50% { transform: scale(1.1); }
          100% { transform: scale(1); }
        }
      
    </style>
  </div>
</DocumentFragment>
`;
